fcrepo-api-x-model/src/main/resources/ontologies/apix-extension.ttl

```diff
@@ -0,0 +1,65 @@
+@prefix apix:<http://fedora.info/definitions/v4/api-extension#> .
+@prefix fcr:<http://fedora.info/definitions/v4/repository#> .
+@prefix owl:<http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:<http://www.w3.org/2000/01/rdf-schema#> .
+@prefix svc:<http://fedora.info/definitions/v4/service#> .
+@prefix xsd:<http://www.w3.org/2001/XMLSchema#> .
+
+apix: a owl:Ontology;
+    owl:imports svc:;
+    rdfs:label "API Extension ontology" .
+
+apix:Extension a owl:Class;
+    rdfs:label "API-X extension"@en;
+    rdfs:comment
+    "Relates Services to Repository objects. May expose a service on repository resources (‘exposing’ modality), or modify request or responses to repository objects (‘intercepting’ modality)."@en .
```

Comment:
No, `.*?` didn't work.  The regex works fine manually.  Enabling debug logging doesn't reveal useful information.  I've confirmed that checkstyle truly is using the modified file, but have no way of confirming that it is actually executing the `RexepSingleline` in the first place, nor a way of confirming if checkstyle is looking at the `.ttl` file in the first place outside of attaching a debugger to see what's really going on.

So I'm just going to fix the smart quotes with a commit, then maybe add a jira issue to add smart quote checking to fcrepo-build-tools